.github/workflows/release.yml

```diff
@@ -17,17 +17,17 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
     steps:
-    - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
-      with:
-        submodules: recursive
+    - name: Checkout sources
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
     - name: Config environment
       if: runner.os == 'Linux'
-      run: sudo apt-get install gcc-multilib
+      run: sudo apt-get update && sudo apt-get install gcc-multilib
     - name: Build C library
-      run: python buildall.py ${{ runner.os != 'macOS' && '-pt -v' || '-v' }}
+      run: python buildall.py -v ${{ runner.os != 'macOS' && '-pt' || '' }}
```

Comment:
Don't we want the release artifacts to contain the Fortran binaries? I would have expected an `-ft` here or something like that. To be sure, I have never used those binaries so I don't mind one way or another, but I do think the previous behavior was that the Fortran binaries were somehow included, no?

Reply:
I agree with you. So I added a build of everything we provide into the release package and included fortran binaries. I think in the future it would be fine to add the Rust and Python packages to the release as well.